src/vm/i386/jitinterfacex86.cpp

```diff
@@ -72,13 +72,37 @@ extern "C" void STDCALL WriteBarrierAssert(BYTE* ptr, Object* obj)
 
 #endif // _DEBUG
 
+#ifdef FEATURE_PAL
+FCDECL3(void, JIT_Stelem_Ref, PtrArray* array, unsigned idx, Object* val)
```

Comment:
THere already are portable implementations of these functions in src/vm/jithelpers.cpp. It seems that all you need to do is to ifdef-out for FEATURE_PAL the following defines at the end of the src/vm/i386/cgencpu.h, the portable versions of these functions will automatically kick in.
```C++
#define JIT_ChkCastClass            JIT_ChkCastClass
#define JIT_ChkCastClassSpecial     JIT_ChkCastClassSpecial
#define JIT_IsInstanceOfClass       JIT_IsInstanceOfClass

#define JIT_Stelem_Ref              JIT_Stelem_Ref
```